src/Validator/SecurityTxtValidator.php

```diff
@@ -45,4 +50,24 @@ public function validate(SecurityTxt $securityTxt): SecurityTxtValidateResult
 		return new SecurityTxtValidateResult($errors, $warnings);
 	}
 
+
+	public function validateWithFetchResult(SecurityTxt $securityTxt, SecurityTxtFetchResult $fetchResult): SecurityTxtValidateResult
+	{
+		$result = $this->validate($securityTxt);
+		try {
+			$this->canonicalUrlValidator->validate($securityTxt, $fetchResult);
```

Comment:
Do not call the `validate` method directly. Use it just like the other validators in that namespace.